### DISCUSSION TITLE
SQL智能助手的工作原理是什么？

### DISCUSSION THREAD

**Discussion by mengbaba3316** [OtterMind/Chat2DB#483] [Q&A]
近期通过一些推文了解到chat2db，感觉sql智能助手应该是其特色，但是对于保密级别较高的项目，使用该功能，会不会照成数据的泄露，chat2db是将那些内容上传给AI呢 @zhuangjiaju

**Comment by JerryFan626** [comment]
我们会上传表的结构，如果对这个敏感的就不能用了。
得私有化部署AI

**Reply by luohc2004** [reply]
我用deepseek chat模型，看来，是提示词里传了表结构，做了提示词优化，返回的还是llm输出，连llm说的话和sql代码一起返回，还要用户自己留下sql代码运行。这样和自己用llm有什么区别呢？自动生成表结构？
